src/voronoi.js

```diff
@@ -46,8 +47,8 @@ function cleanData(data){
                 i !== j &&
                 Math.abs(d[0] - e[0]) === Math.abs(d[1] - e[1])
             ){
-                d[0] = d[0] + 1e-10*d[1];
-                d[1] = d[1] + 2e-10*d[0];
+                d[0] = d[0] + epsOffset;
+                d[1] = d[1] + epsOffset;
```

Comment:
Maybe instead of adding the constant value here we can subtract it?

In a situation where the points are on a square, adding the same value will mean that they are still on a square. 

`d[1] = d[1] - epsOffset;`

Reply:
Applied changes, not sure about changing sign of offset since that will just hit square this another points orientation.
Works with my data sets both ways.
Committed auto-generated changes in package-lock.json.